server/sonar-process/src/main/java/org/sonar/process/cluster/ClusterConstants.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.process.cluster;/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+public interface ClusterConstants {
+  String OPERATIONAL_PROCESSES = "OPERATIONAL_PROCESSES";
+  String LEADER = "LEADER";
+  String HOSTNAME = "HOSTNAME";
+  String SONARQUBE_VERSION = "SONARQUBE_VERSION";
+  String CLIENT_UUIDS = "CLIENT_UUIDS";
+  String WORKER_UUIDS = "WORKER_UUIDS";
```

Comment:
Since the cluster is global if someone is using the exact same key, it will broke the cluster.
Obviously, we can use specific key for each process (org.sonar.ce.WORKER_UUIDs) to not mess up between usage of Hazelcast between processes.
I've found adding package prefix a bit too overengineering right now since we only have 6 properties for the time being.